content/en/blog/releases/v0.6.0.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Build v0.6.0"
+date: 2021-10-11T16:00:00-04:00
+draft: false
+author: "Sascha Schwarze ([@SaschaSchwarze0](https://github.com/SaschaSchwarze0))"
+---
+
+It was a long journey, a whole summer actually, and more than it should have been taken, but finally, here it is: v0.6.0 of Shipwright, our first release after having joined the [Continuous Delivery Foundation](https://cd.foundation/).
+
+The feature list is impressive. It made it hard to pick some top 3. Here is my choice:
+
+## Parameterization of Builds
+
+Our API objects; the build strategy that defines the steps to turn source code into a container image, the build where a user configures his container image build, and the build run as an instance of a build; so far suffered of a major capability: parameterization. We defined the available inputs that a user specified in a build. Those were only the minimum: the location of the source code, the name of the strategy and the destination image. Any other customization of the build was not possible, limiting the possible scenarios.
+
+Two major enhancements bring us a huge step forward:
+
+Build strategy authors can now define parameters that they can use then in their steps. Build users provide the values for those parameters. Our sample build strategies start to make use of this feature: the [BuildKit sample](https://github.com/shipwright-io/build/blob/v0.6.0/samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml#L14-L16) now contains a parameter to enable the usage of insecure container registry, and the [ko sample](https://github.com/shipwright-io/build/blob/v0.6.0/samples/buildstrategy/ko/buildstrategy_ko_cr.yaml#L8-L22) comes with a rich set of parameters to select Go version and flags, ko version and the main package directory. The [ko sample build](https://github.com/shipwright-io/build/blob/v0.6.0/samples/build/build_ko_cr.yaml#L10-L15) outlines how those parameters can be used to build the Shipwright Build controller image. Read more about parameters in the documentation for [build strategy authors](https://github.com/shipwright-io/build/blob/v0.6.0/docs/buildstrategies.md#strategy-parameters) and [build users](https://github.com/shipwright-io/build/blob/v0.6.0/docs/build.md#defining-paramvalues).
```

Comment:
It would probably be better to link to just the yaml file, instead of specific line numbers, as those will change over time.

Reply:
This links to line numbers: `[BuildKit sample](https://github.com/shipwright-io/build/blob/v0.6.0/samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml#L14-L16)`

Reply:
Right, sry, better explanation: I think it is fine to link to the line number because I referenced the v0.6.0 tag which will never change. We definitely should not link to line numbers in main, but that's not happening here.